File: frontend/src/model/Project.ts
import { ImageClass } from "./ImageClass";
import { Models } from "./Models";

export class Project {
  public idProject: number;
  public name: string;
  public date: string;
  public description: string;
  public idUser: string;
  public classes: Array<ImageClass>;
  public models: Array<Models>;

  constructor(name: string, date: string, description: string, idUser: string) {
    this.idProject = -1;
    this.name = name;
    this.date = date;
    this.description = description;
    this.idUser = idUser;
    this.classes = [];
    this.models = [];
  }

  getIdProject() {
    return this.idProject;
  }
}

File: backend/backend/Models/Project.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace backend.Models
{
    public partial class Project
    {
        public Project()
        {
            Classes = new HashSet<Class>();
            Data = new HashSet<Datum>();
            Members = new HashSet<Member>();
            Models = new HashSet<Model>();
        }

        public long IdProject { get; set; }
        public string Name { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; }
        public string IdUser { get; set; }

        public virtual CUser IdUserNavigation { get; set; }
        public virtual ICollection<Class> Classes { get; set; }
        public virtual ICollection<Datum> Data { get; set; }
        public virtual ICollection<Member> Members { get; set; }
        public virtual ICollection<Model> Models { get; set; }
    }
}
